server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -121,9 +121,7 @@ export class ProtocolOpHandler implements IProtocolHandler {
 				break;
 
 			case MessageType.Propose:
-				// This should become unconditional once (Loader LTS) DeltaManager.processInboundMessage() stops parsing content (ADO #12052)
-				// Note: Until that change is made in the loader, this case will never be hit.
-				// Then there will be a long time of needing both cases, until LTS catches up to the change.
+				// TODO: Update callers to stop parsing the contents and do it here unconditionally
 				if (typeof message.contents === "string") {
```

Comment:
Feels like another place where we can remove IF and make it unconditional. If loader does not parse it, this code has to parse it (otherwise it will break).